cogs/help.py

```diff
@@ -30,10 +30,12 @@
 HELP_LOGS_CHANNEL_ID: int = int(env["HELP_LOG_CHANNEL_ID"])
 HELPER_ROLE_ID: int = int(env["HELP_NOTIFICATION_ROLE_ID"])
 HELP_MOD_ID: int = int(env["HELP_MOD_ROLE_ID"])
+NO_HELP_ID: int = int(env["NO_HELP_ROLE_ID"])
```

Comment:
Could this be renamed to `HELP_BANNED_ID` just to be more consistent with the Nextcord server role names.

Reply:
`HELP_BAN_ID` hehe
SCREAMING SNAKE CASE ftw,

Reply:
> `HELP_BAN_ID` hehe SCREAMING SNAKE CASE ftw,

🙏🏼